core/client/src/main/java/alluxio/client/netty/NettyRemoteBlockReader.java

```diff
@@ -71,7 +79,7 @@ public ByteBuffer readRemoteBlock(InetSocketAddress address, long blockId, long
           .writeAndFlush(new RPCBlockReadRequest(blockId, offset, length, lockId, sessionId));
       channelFuture = channelFuture.sync();
       if (channelFuture.isDone() && !channelFuture.isSuccess()) {
-        LOG.error("Failed to write to %s for block %d with error %s.", address.toString(), blockId,
+        LOG.error("Failed to write to %s for block %d with error %s.", mAddress.toString(), blockId,
```

Comment:
is this `LOG` message correctly using `%s` rather than `{}`? Feel free to address this later to keep this  PR clean